modules/ducktests/tests/ignitetest/services/postgresql/postgresql.py

```diff
@@ -0,0 +1,196 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains classes and utilities to start PostgreSql cluster for testing CDC.
+"""
+
+import os.path
+from distutils.version import LooseVersion
```

Comment:
Please change to 
`from looseversion import LooseVersion`

Otherwise it doesn't work in docker (which contains python 3.12 now) with error

`
[ERROR:2025-10-15 12:59:25,616]: Failed to import ignitetest.tests.smoke_test, which may indicate a broken test that cannot be loaded: ModuleNotFoundError: No module named 'distutils'
`